odl/discr/lp_discr.py

```diff
@@ -1336,35 +1279,17 @@ def uniform_discr_fromdiscr(discr, min_pt=None, max_pt=None,
     ----------
     discr : `DiscreteLp`
         Uniformly discretized space used as a template.
-    min_pt : float or sequence of floats, optional
-        Minimum corners of the desired function domain.
-    max_pt : float or sequence of floats, optional
-        Maximum corners of the desired function domain.
+    min_pt, max_pt: float or sequence of floats, optional
+        Desired minimum/maximum corners of the new space domain.
     shape : int or sequence of ints, optional
-        Number of samples per axis.
-    cell_sides : array-like, optional
```

Comment:
Ah, I moved all the deleted stuff into `kwargs` and just mention the pass-through below.